sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -73,6 +73,17 @@ class DatasetSuite extends QueryTest with SharedSQLContext {
       ("a", 2), ("b", 3), ("c", 4))
   }
 
+  ignore("Dataset should set the resolved encoders internally for maps") {
```

Comment:
because it could get optimized away when we introduce local execution mode and better pipelining.